src/link/index.tsx

```diff
@@ -14,6 +17,21 @@ const Link = React.forwardRef(
     const baseComponentProps = useBaseComponent('Link', {
       props: { color, external, fontSize, rel: props.rel, target: props.target, variant: props.variant },
     });
+
+    const analyticsMetadata: GeneratedAnalyticsMetadataLinkFragment = {
+      action: 'click',
+      detail: {
+        label: '',
+        href: props.href || '',
+        external: `${external}`,
+      },
+      component: {
+        name: 'awsui.Link',
+        label: '',
+        properties: { variant: props.variant || 'secondary' },
```

Comment:
If `secondary` is our default variant, can we resolve it in a single place instead of passing a fictional value here?

Reply:
The default variant is a bit tricky, as it depends on a React Context. I decided to keep this simpler. https://github.com/cloudscape-design/components/blob/main/src/link/internal.tsx#L59

Reply:
This way you will not be able to tell apart explicit "secondary" variant and assumed value. Up to you, but the current reported value cannot be used for any decisions, IMO